index.html

```diff
@@ -15,37 +15,68 @@
-            <a class="dropdown-item" href="#">Something else here</a>
-          </div>
-        </li>
-      </ul>
+  <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+    <div class="container">
+      <a class="navbar-brand" href="#">JavaSip</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+      <div class="collapse navbar-collapse" id="navbarNavDropdown">
+        <ul class="navbar-nav">
+          <li class="nav-item active">
+            <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Features</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Pricing</a>
+          </li>
+          <li class="nav-item dropdown">
+            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+              Dropdown link
+            </a>
+            <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+              <a class="dropdown-item" href="#">Action</a>
+              <a class="dropdown-item" href="#">Another action</a>
+              <a class="dropdown-item" href="#">Something else here</a>
+            </div>
+         </li>
+        </ul>
+      </div>
     </div>
-  </div>
-</nav>
+  </nav>
+
+  <!-- age verification -->
+
+  <div class="container row">
+      <div class="col">
+          <div class="age-checker-content">
+            <h1>We need to check your ID</h1>
+              <p>You must be of legal drinking age to enter this site</p>
+          </div>
+
+            <form class="age-checker-form" method="GET">
+              <div class="row">
+                  <div class="date-field col-xs-4">
+                      <input id="month" type="number" inputmode="numeric" pattern="[0-9]*" name="month" placeholder="MM"   min="1" max="12" maxlength="2">
+                  </div>
+                  <div class="date-field col-xs-4">
+                      <input id="day" type="number" inputmode="numeric" pattern="[0-9]*" name="day" placeholder="DD" min="1" max="31" maxlength="2">
+                  </div>
+                  <div class="date-field col-xs-4">
+                      <input id="year" type="number" inputmode="numeric" name="year" placeholder="YYYY"  min="1900" max="2100" maxLength="4" pattern="[0-9]*">
+                  </div>
+              </div>
+            <div class="row">
+                <div class="col-xs-12">
+                    <button id="age-check">ENTER</button>
+                </div>
+            </div>
+          </form>
+      </div> 
+
+      <div class="modal"> 
```

Comment:
Move `<div class="modal">` on L79 to above the age verification as we want to create a modal container that can span the page. In doing this, also delete the two `</div>` added on L133 and L134 and move one of them below the age verification code you just added to close out this container.

Reply:
Now that we have a modal in place you will need to add CSS for the #modal container. Think: 

`background-color` so it hides the content underneath
`min-height` so it spans the height of the page
`min-width` so it spans with width of the page
`z-index` so it appears on top of all other content
`position` so you can control where on the page it is located